### DISCUSSION TITLE
I am unable to use p:printer to print anything. How can I resolve this issue?

### DISCUSSION THREAD

**Discussion by liu-will** [primefaces/community#191] [PrimeFaces Templates]
I purchased the Siberia Layout and did development work, but I found that I cannot use p:printer. When I invoke p:printer for browser preview, it displays a blank page. Additionally, when I use Ctrl+P, all pages appear blank. Why is this happening, and how can I resolve it?

**Comment by NicolaIsotta** [comment]
Already reported for Apollo, I think it's the same issue: https://forum.primefaces.org/viewtopic.php?f=41&t=76082

**Reply by liu-will** [reply]
Thank you for the response. When I entered the Live Demo to check, I also noticed that when using Ctrl+P to preview, it was also showing a blank page. Upon inspecting the Siberia source code, I found that the 'invoice' page is using 'window.print()' instead of 'p:printer'. Currently, I have downgraded the version to PrimeFaces 11.0.0, and I can use 'p:printer' normally. So, I suspect it might be an issue with my development environment. My development environment is using JDK 1.8.

**Comment by mertsincan** [comment]
Hi,
Could you please try the latest Siberia version? For now, closed. If the problem persists, please reopen this ticket. Thanks a lot for your understanding!

Best Regards,
